sdk/common/src/test/java/io/opentelemetry/sdk/resources/ResourceTest.java

```diff
@@ -330,4 +331,45 @@ public void toBuilder() {
     assertThat(newResource.getAttribute(stringKey("foo"))).isEqualTo("val");
     assertThat(newResource.getSchemaUrl()).isEqualTo("http://example.com");
   }
+
+  @Test
+  public void removeIf() {
+    ResourceBuilder builder = Resource.builder();
+    assertThat(builder.removeIf(unused -> true)).isEqualTo(builder);
+    assertThat(Resource.builder().removeIf(key -> key.getKey().equals("key1")).build())
+        .isEqualTo(builder.build());
+    assertThat(
+            Resource.builder()
+                .put("key1", "value1")
+                .removeIf(key -> key.getKey().equals("key1"))
+                .removeIf(key -> key.getKey().equals("key1"))
+                .build())
+        .isEqualTo(Resource.builder().build());
```

Comment:
```suggestion
    assertThat(Resource.builder().removeIf(unused -> true)).isEqualTo(Resource.empty());
    assertThat(Resource.builder().removeIf(key -> key.getKey().equals("key1")).build())
        .isEqualTo(Resource.empty());
    assertThat(
            Resource.builder()
                .put("key1", "value1")
                .removeIf(key -> key.getKey().equals("key1"))
                .removeIf(key -> key.getKey().equals("key1"))
                .build())
        .isEqualTo(Resource.empty());
```

Reply:
Great thanks! I have corrected it as requested.